mlir/include/mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h

```diff
@@ -230,19 +255,17 @@ class PopulateParamsXDL : public BasePopulateParams<InitParamsXDL> {
   // if can't select config from above , use this config to do
   // padding kernel for example , GEMMK/block is 16 , if your gemmK is  13 , we
   // add more 3 gemmk.
-  static const InitParamsXDL universalParameters;
-
   uint32_t obtainBlockSize(const InitParamsXDL &params, int64_t waveSize);
 
-  LogicalResult getKBlocks(Conv2DBwdWeightOp op, const GemmSize &gemmSize,
-                           const InitParamsXDL &params, int64_t &gemmKBlocks);
+  LogicalResult getKBlocks(int64_t convGroupSize, const GemmSize &gemmSize,
```

Comment:
This should be named `convBatchSize`?

Reply:
I called `batchSize` to be coherent with the rest.